Dockerfile

```diff
@@ -0,0 +1,17 @@
+# syntax=docker/dockerfile:1
+
+FROM python:3
+
+WORKDIR /mysite
+
+COPY requirements.txt requirements.txt
+RUN pip3 install -r requirements.txt
+
+COPY /mysite .
+
+RUN python manage.py migrate
+RUN DJANGO_SUPERUSER_PASSWORD=demo python manage.py createsuperuser --noinput --username demo --email demo@mail.com
```

Comment:
As a general security measure, we don't generate super user in the Dockerfile. Because otherwise everyone will have super user privilage.

Put it in the README.md as an initial setup guide. If you have multiple scripts need to run initially, group them with `Makefile`.